<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignInWindow</class>
 <widget class="QMainWindow" name="SignInWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: rgb(139, 94, 60);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;</string>
     </property>
     <property name="text">
      <string>Already have Account? </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/img/img/Frame 245.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="GoToLogInButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>-1</pointsize>
       <italic>true</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton {
    background-color: #6A442B; /* Brown color */
    color: white;                 
    border-radius: 20px;               /* Adjust for pill shape */
                   /* Vertical padding, width will be set by layout */
    font-weight: bold;                 /* Bold text */
    font-size: 16px;                   

}</string>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <italic>true</italic>
      <fontweight>DemiBold</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color : rgb(62, 47, 28);
}</string>
    </property>
    <property name="text">
     <string>Sign Up </string>
    </property>
    <property name="textFormat">
     <enum>Qt::TextFormat::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>56</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Please provide your information </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="SignInButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>-1</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
    background-color: rgb(139, 94, 60); /* Brown color */
    color: white;                 
    border-radius: 20px;               /* Adjust for pill shape */
                   /* Vertical padding, width will be set by layout */
    font-weight: bold;                 /* Bold text */
    font-size: 16px;                   

}</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>17</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/img/img/SvgjsG2242.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>231</width>
      <height>193</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="FirstNameEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;  
color: rgb(62, 47, 28);            /* Add some internal padding */
}

QLabel {
    color: #555555; /* A darker grey */
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>First Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LastNameEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Last Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="EmailEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;  
color: rgb(62, 47, 28);            /* Add some internal padding */
}

QLabel {
    color: #555555; /* A darker grey */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>EMAIL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UniversityEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>UNIVERSITY</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLineEdit" name="IDedit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>ID CARD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">QLabel
{
background :rgb(255, 255, 255);
 border-radius: 10px; 
}</string>
         </property>
         <property name="text">
          <string>Date Of Birth</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="DateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="RFIDEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>RFID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="TypeComboBox">
         <property name="placeholderText">
          <string>TYPE</string>
         </property>
         <item>
          <property name="text">
           <string>student</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>teacher</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>administrator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>librarian</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>externel</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Library Director</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLineEdit" name="PasswordEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PasswordConfEdit">
         <property name="styleSheet">
          <string notr="true">QLineEdit {
    border: 1px solid #CFCFCF; /* A light grey border */
    border-radius: 10px;       /* Adjust this value for desired roundness */
    padding: 5px;              /* Add some internal padding */
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>Password confirmation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
